### PR TITLE
Deprecated `add_data_qubit` to `add_qubit` [unitaryhack]

### LINKED ISSUE #136: Deprecate `add_data_qubit` to `add_qubit`
Currently, the Host method `add_data_qubit` is named poorly. We should rename it to just `add_qubit`.

### PR DESCRIPTION
**Pull request by divshacker**
fixes #136

### CODE REVIEW

File: qunetsim/components/host.py [hunk 1]
```diff
@@ -1341,6 +1341,12 @@ def add_data_qubit(self, host_id, qubit, q_id=None):
         Returns:
             (str): The qubit ID
         """
+        warnings.warn(
```
Comment (stephendiadamo) [inline qunetsim/components/host.py:1344]: I see the deprecation message, but. I don't see the new method called `add_qubit`. It seems to be missing... Do the tests pass with this code?

**Review by stephendiadamo — CHANGES_REQUESTED** [review]
I see the deprecation message, but I don't see the new `add_qubit` method added to the Host component. It needs to be added.

**Comment by divshacker** [comment]
I have added the function, Please check now and let me know further changes

**Review by SimonSekavcnik — APPROVED** [review]
Approved.

**Comment by divshacker** [comment]
Took ten months to test and approve ? 🙂

**Comment by SimonSekavcnik** [comment]
Sorry about that.

**Review by stephendiadamo — APPROVED** [review]

**Pull request merged**